bgpd/bgpd.h

```diff
@@ -130,6 +130,7 @@ struct bgp_master {
 #define BGP_OPT_MULTIPLE_INSTANCE        (1 << 1)
 #define BGP_OPT_CONFIG_CISCO             (1 << 2)
 #define BGP_OPT_NO_LISTEN                (1 << 3)
+#define BGP_OPT_EMULATE_VRF              (1 << 4)
```

Comment:
Wouldn't it be better to rename this to `BGP_OPT_NO_ZEBRA`?

Reply:
ok